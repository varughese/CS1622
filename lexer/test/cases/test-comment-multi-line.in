aaa /* not visible
to the
scanner because */ bbb /* i am 
a comment*/ccc/**/ddd
