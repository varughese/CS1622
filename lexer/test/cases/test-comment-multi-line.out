(1,ID,"aaa")
(3,ID,"bbb")
(4,ID,"ccc")
(4,ID,"ddd")
